### ISSUE TITLE
Rubocop/Style/HashTransformValues has extranous output that breaks tooling

### ISSUE THREAD

**Issue by olbrich** [rubocop-hq/rubocop#7768]
When running rubocop 0.80.0 I see a spurious `.CCC...RuboCop::AST::BlockNode
C.C..` in the output.  This also causes json formatted output to be invalid because it prefixes `RuboCop::AST::BlockNode` before the start of the JSON string.

Seems to be related to `Rubocop/Style/HashTransformValues`

```
engine produced invalid output: {:message=>"Invalid JSON", :output=>"RuboCop::AST::BlockNode\n{\"type\":\"Issue\",\"check_name\":\"Rubocop/Style/HashTransformValues\",\"description\":\"Prefer `transform_values` over `each_with_object`.\",\"categories\":[\"Style\"],\"remediation_points\":50000,\"location\":{\"path\":\"app/decorators/conflict_decorator.rb\",\"positions\":{\"begin\":{\"column\":5,\"line\":88},\"end\":{\"column\":8,\"line\":90}}},\"content\":{\"body\":\"This cop looks for uses of `_.each_with_object({}) {...}`,\\n`_.map {...}.to_h`, and `Hash[_.map {...}]` that are actually just\\ntransforming the values of a hash, and tries to use a simpler \\u0026 faster\\ncall to `transform_values` instead.\\n\\nThis can produce false positives if we are transforming an enumerable\\nof key-value-like pairs that isn't actually a hash, e.g.:\\n`[[k1, v1], [k2, v2], ...]`\\n\\nThis cop should only be enabled on Ruby version 2.4 or newer\\n(`transform_values` was added in Ruby 2.4.)\\n\\n### Example:\\n    # bad\\n    {a: 1, b: 2}.each_with_object({}) { |(k, v), h| h[k] = foo(v) }\\n    {a: 1, b: 2}.map { |k, v| [k, v * v] }\\n\\n    # good\\n    {a: 1, b: 2}.transform_values { |v| foo(v) }\\n    {a: 1, b: 2}.transform_values { |v| v * v }\"}}"}
```
--------

## Expected behavior

No extraneous output.

## Actual behavior

Extra output

## Steps to reproduce the problem

Run rubocop with Rubocop/Style/HashTransformValues enabled.

## RuboCop version

```
0.80.0 (using Parser 2.7.0.3, running on ruby 2.6.5 x86_64-darwin18)
```

**Comment by koic** [comment]
This issue fixed by #7734. It will be resolved in the next RuboCop release. Thanks!
